hibike/lib/hibike/hibike_device.cpp

```diff
@@ -2,19 +2,27 @@
 
 message_t hibikeBuff;
 uint16_t params, old_params, subDelay;
-uint32_t value, disable_latency, heartbeatDelay, prevHeartTimer;
-uint64_t prevSubTime, currTime, sent_heartbeat, resp_heartbeat;
+
+//Heartbeat settings.  
+uint32_t disable_latency, heartbeat_Delay;
```

Comment:
if you're going to be renaming the variables, could you standardize the capitalization in `heartbeat_Delay`. Either camelCase or snake_case is fine, but a mix seems odd.